substrate/client/utils/src/mpsc.rs

```diff
@@ -68,25 +68,25 @@ impl<T> Clone for TracingUnboundedSender<T> {
 #[derive(Debug)]
 pub struct TracingUnboundedReceiver<T> {
 	inner: Receiver<T>,
-	name: &'static str,
+	name: String,
 }
 
 /// Wrapper around [`async_channel::unbounded`] that tracks the in- and outflow via
 /// `UNBOUNDED_CHANNELS_COUNTER` and warns if the message queue grows
 /// above the warning threshold.
 pub fn tracing_unbounded<T>(
-	name: &'static str,
```

Comment:
Since 1.72 you can use `String::leak()` https://doc.rust-lang.org/nightly/alloc/string/struct.String.html#method.leak